Redis/redis_config.py

```diff
@@ -0,0 +1,16 @@
+import config
```

Comment:
nit: `redis` (not `Redis`) in the file path

Reply:
Changing it to redis would conflict with the package redis. I've renamed it redis_interface to keep them from conflicting.